homeassistant/components/youless/sensor.py

```diff
@@ -58,8 +58,8 @@ class YouLessSensorEntityDescription(SensorEntityDescription):
         key="gas",
         device_group="gas",
         device_group_name="Gas meter",
-        name="Gas usage",
-        icon="mdi:fire",
+        translation_key="total_gas_m3",
```

Comment:
Also, there are device class translations, so if you have an entity which has the same context and name as the device class, you can omit the translation key and it will take the device class name

Reply:
I updated the integration to use the device_group as the identifier for the translation of the device. That way the name no longer needs to be in the devicedescription and can be localized.